@ecomplus/storefront-app/src/components/html/EcPayment.html

```diff
@@ -78,7 +78,9 @@
         </small>
       </a>
 
-      <fade-transition>
+      <transition
+        enter-active-class="animated fadeInDown"
+        leave-active-class="animated fadeInUp">
```

Comment:
```suggestion
        leave-active-class="animated fadeOutUp">
```

Fade in no leave daria o efeito oposto ao que queremos...